app/assets/javascripts/client/views/directives/start_assessment.html

```diff
@@ -1,11 +1,12 @@
 <div>
-  <div class="row start-assessment plus-button">
+  <div class="start-assessment plus-button">
       <a href="" data-toggle="modal" data-target="#startAssessment">
-        <div class="col-sm-1 col-md-1">
-          <i class="fa fa-plus"></i>
-        </div>
-        <div class="col-sm-11 col-md-11 heading">
-          <h1>{{text()}}</h1>
+        <span class="fa-stack">
+          <i class="fa fa-circle fa-stack-2x"></i>
+          <i class="fa fa-plus fa-stack-1x"></i>
+        </span>
+        <div class="heading">
+          <a>{{text()}}</a>
```

Comment:
This anchor tag interferes with the parent anchor tag which drives the modal. What that results in is modal not opening when the text is clicked, but only when the icon is clicked.

Removing this anchor tag would suffice.